shard_to_single_vcf.py

```diff
@@ -0,0 +1,65 @@
+## THIS SCRIPT NEEDS SSDs!!!!
+import argparse
+
+import hail as hl
+from gnomad.utils.slack import try_slack
+
+
+def main(args):
+
+    hl.init(log="/vcf_write.log", default_reference=f"{args.build}")
+
+    mt = hl.import_vcf(
+        f"{args.genotypes_vcf}", force_bgz=True, find_replace=("nul", ".")
+    )
+    filters_ht = hl.import_vcf(
+        f"{args.filtered_vcf}", force_bgz=True, find_replace=("nul", ".")
+    ).rows()
+    mt = mt.annotate_rows(filters=filters_ht[mt.row_key].filters)
```

Comment:
I added some counts when I ran this out of curiosity...not sure if they're a good idea to add? They might make the code kind of slow as the callsets get bigger

Reply:
Yeah, I think that would trigger another pass through the data?